src/components/Select/Select.js

```diff
@@ -26,7 +26,9 @@ import {
 } from '../../util/shared-prop-types';
 import { textMega, disableVisually } from '../../styles/style-helpers';
 
-import { ReactComponent as ArrowsIcon } from './arrows.svg';
+import { ReactComponent as ArrowsIcon } from '../../icons/arrows.svg';
```

Comment:
I think we need to consider adding support for some shorthand aliases like `~/` with https://github.com/tleunen/babel-plugin-module-resolver as we did on the website.

Reply:
I agree in principle, but I'd create a proposal issue first to get consensus from the rest of the web chapter.

Reply:
absolutely!